### PR TITLE
Some changes/improvements to german translation

### PR DESCRIPTION
**Pull request by porky11**

### CODE REVIEW

**Comment by Jikstra** [comment]
Thank you, very appreciated!
Probably we need to push the to deltachat-android, so that they get synced with transifex. @r10s how is the current situation with translations?

**Comment by r10s** [comment]
> Probably we need to push the to deltachat-android, so that they get synced with transifex. @r10s how is the current situation with translations?

no need to push to android, translations but english needs to be changed on transifex, you can login with your github account there, see https://delta.chat/en/contribute#translations-and-bug-reports

for the other suggestions, cc @Yettte

**Comment by dotlambda** [comment]
> to the changes: afaik, the hyphen is not needed for "Delta Chat" unless it is an compound word, so, "Delta Chat" but "Delta-Chat-App", not sure, though.

I would say "Delta Chat-App".

**Comment by r10s** [comment]
> I would say "Delta Chat-App".

hm, that's probably not so fortunate as that glues togehter "Chat-App", leaving "Delta" alone, in general, that seems not to be the goal of the hyphen in german, cmp. https://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich

that in mind, the current spelling "Delta Chat App" seems to be just fine, see examples at https://de.wikipedia.org/wiki/Durchkopplung#Eigennamen_von_Institutionen

anyway, i suggest to close this pr as it is not actionable here.

File: _locales/de.xml [hunk 1]
```diff
@@ -410,7 +410,7 @@
     <string name="tab_map">Karte</string>
     <string name="tab_gallery_empty_hint">In diesem Chat geteilte Bilder und Videos werden hier angezeigt.</string>
     <string name="tab_docs_empty_hint">In diesem Chat geteilte Dokumente, Musik und Dateien werden hier angezeigt.</string>
-    <string name="media_preview">Media-Vorschau</string>
+    <string name="media_preview">Medien-Vorschau</string>
```
Comment (r10s) [inline _locales/de.xml:413]: i applied this change to transifex.

File: _locales/de.xml [hunk 2]
```diff
@@ -446,11 +446,11 @@
     <string name="import_backup_ask">Backup \"%1$s\" gefunden.\n\nDieses einlesen und alle Daten und Einstellungen daraus verwenden?</string>
     <string name="import_backup_no_backup_found">Kein Backup gefunden.\n\nDie Sicherung nach \"%1$s\" kopieren und es erneut versuchen. Alternativ \"Jetzt beginnen\" wählen, um den normalen Einrichtungsprozess fortzusetzen.</string>
     <!-- Translators: %1$s will be replaced by the e-mail address -->
-    <string name="login_error_cannot_login">Login als \"%1$s\" nicht möglich. Bitte E-Mail Adresse und Passwort auf Richtigkeit überprüfen.</string>
+    <string name="login_error_cannot_login">Login als \"%1$s\" nicht möglich. Bitte E-Mail-Adresse und Passwort auf Richtigkeit überprüfen.</string>
```
Comment (r10s) [inline _locales/de.xml:449]: i applied this change to transifex.

File: _locales/de.xml [hunk 3]
```diff
@@ -496,7 +496,7 @@ Einige Anbieter platzieren zusätzliche Informationen im Posteingang, die Sie z.
     <string name="pref_vibrate">Vibrieren</string>
     <string name="pref_screen_security">Bildschirmsicherheit</string>
     <!-- Translators: The wording must indicate that we can't guarantee that, its a System flag that we set. But the System in question must honor it. -->
-    <string name="pref_screen_security_explain">Aufforderung zum Sperren von Screenshots in der Kürzlich erstellten Liste und in der App</string>
+    <string name="pref_screen_security_explain">Aufforderung zum Sperren von Screenshots in der kürzlich erstellten Liste und in der App</string>
```
Comment (r10s) [inline _locales/de.xml:499]: i applied this change to transifex.

File: _locales/de.xml [hunk 4]
```diff
@@ -523,11 +523,11 @@ Einige Anbieter platzieren zusätzliche Informationen im Posteingang, die Sie z.
     <string name="pref_read_receipts_explain">Wenn Lesebestätigungen ausgeschaltet sind, werden auch keine Lesebestätigungen von anderen angezeigt. </string>
     <string name="pref_manage_keys">Schlüssel verwalten</string>
     <string name="pref_use_system_emoji">System-Emoji benutzen</string>
-    <string name="pref_use_system_emoji_explain">Eingebaute Emoji-Unterstützung von Delta Chat deaktivieren</string>
+    <string name="pref_use_system_emoji_explain">Eingebaute Emoji-Unterstützung von Delta-Chat deaktivieren</string>
     <string name="pref_app_access">App-Zugriff</string>
     <string name="pref_communication">Kommunikation</string>
     <string name="pref_chats">Chats</string>
-    <string name="pref_in_chat_sounds">In-Chat Klänge</string>
+    <string name="pref_in_chat_sounds">In-Chat-Klänge</string>
```
Comment (r10s) [inline _locales/de.xml:530]: i applied this change to transifex.

**Review by r10s — CHANGES_REQUESTED** [review]
i applied the typo fixes to transifex - thanks a lot for taking care, @porky11. the changes will apply to all apps on the next pull of translations then.

as said above, i suggest to close this as there is no actionable item left here. of course, discussion can continue.

**Pull request closed without merging**